app/code/local/Etech/Mydata/Model/Profile.php

```diff
@@ -1,26 +1,23 @@
 <?php
 
-class Etech_Mydata_Model_Profile extends Mage_Core_Model_Abstract
-{
-    public function _construct()
-    {
+class Etech_Mydata_Model_Profile extends Mage_Core_Model_Abstract {
+
+    public function _construct() {
         parent::_construct();
         $this->_init('mydata/profile');
     }
-    
-     public function profileData() {
 
-        $model = $this->getCollection();
-       
-        return $model;
-    }
-    
-    
-    
-     public function lastData($post) {
+    public function lastData($post) {
```

Comment:
sir lastData()  insert the value in profile table. like  first name and last name and last insert id .